### DISCUSSION TITLE
How to use `clickToCopy` with row onClick event

### DISCUSSION THREAD

**Discussion by Abdullah0991** [KevinVandy/mantine-react-table#30] [Q&A] [answered]
Hi,

I am using the `mantineTableBodyRowProps.onClick` event to navigate the user to the details page of the clicked row.
Now I want to enable the `clickToCopy` feature, how to prevent firing the `onClick` event when trying to copy cell content?

**Comment by KevinVandy** [comment, marked as answer]
It is a bug in the current beta that the copy button onclicks did not include stopPropogation by default. That will be fixed in the next beta
